go.mod

```diff
@@ -50,12 +50,12 @@ require (
 	github.com/opencontainers/runc v1.1.12
 	github.com/opencontainers/selinux v1.11.0
 	github.com/pkg/errors v0.9.1
-	github.com/pmezard/go-difflib v1.0.0
+	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
```

Comment:
is it intentional this is pulling in a version between tags? do we know if this library expects to be used on a non-tagged version and is stable there?

Reply:
Sorry, I didn't notice the two non-tag dependencies added.
I investigated those dependencies and found those two dependencies used in `github.com/spf13/viper@v1.17.0`.

https://github.com/spf13/viper/blob/v1.17.0/go.mod#L32
https://github.com/spf13/viper/blob/v1.17.0/go.mod#L61

I think kustomize is still working from those packages that are upgraded due to all tests being passed.
But if you have concerns to make effect other things from the update of those dependencies, I think I can consider downgrading the `github.com/spf13/viper/` package, too.